examples/inverse/plot_vector_mne_solution.py

```diff
@@ -28,14 +28,14 @@
 fname_inv = data_path + '/MEG/sample/sample_audvis-meg-oct-6-meg-inv.fif'
 inv = read_inverse_operator(fname_inv)
 
-# Apply inverse solution, set pick_ori='full' to obtain a VectorSourceEstimate
-# object
+# Apply inverse solution, set pick_ori='vector' to obtain a
+# VectorSourceEstimate object
```

Comment:
(you can cross-check using `make html_dev-pattern` before and after my suggested change)